openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -212,6 +213,17 @@ def change_activation_status(self, activation_status, ids):
         return Response(status=status.HTTP_404_NOT_FOUND)
 
 
+class CourseEnrolmentViewSet(viewsets.ModelViewSet):
+    """
+    Course view-set for bulk enrolment task
+    """
+    def enroll_users(self, request, *args, **kwargs):
+        if request.data.get("ids") and request.data.get("course_keys"):
```

Comment:
can we add check here if users_ids does not belong to same organization then it should raise error